validate.go

```diff
@@ -71,6 +134,9 @@ func validateAndAdjustContainerResourceLimits(container *corev1.Container, setti
 		}
 	}
 
+	if settings.Cpu.IgnoreValues {
+		return false, nil
+	}
```

Comment:
Now that you have other functions that is checking the resource fiel before calling this function, you do not need to check the IgnoreValues again. You can remove this if or remove the new functions.